packages/webview_flutter/CHANGELOG.md

```diff
@@ -1,3 +1,27 @@
+## 0.3.24
+
+* Add support for building `WebView` widget with Android hybrid views. To use this feature, set
```

Comment:
Android hybrid views should have link to https://github.com/flutter/flutter/wiki/Hybrid-Composition